src/toil/jobStores/abstractJobStore.py

```diff
@@ -79,6 +79,7 @@ def __init__(self, config=None):
             config.workflowID = str(uuid4())
             self.__config = config
             self.writeConfigToStore()
+        logger.info('The workflow ID is: %s' %self.__config.workflowID)
```

Comment:
No i believe this should be INFO such that the user is always told what the workflow ID is. They will need it to access their data in the `toil-<workflowID>` folder